88.md

```diff
@@ -0,0 +1,79 @@
+NIP-88
+======
+
+Recurring Subscriptions
+-----------------------
+
+`draft` `optional` `author:pablof7z`
+
+This NIP defines a way for a pubkey to create recurring subscription payments to another pubkey.
+
+## Tier Event
+A pubkey can create "tiers". These tiers might provide certain benefits to the supporters who subscribe to these.
+
+```json
+{
+    "kind": 7002,
+    "content": "<description of the tier>",
+    "tags": [
+        [ "title", "..." ],
+        [ "amount", "<amount-in-base-unit>", "currency", "<cadence>" ]
```

Comment:
In NIP-99 we use the same tag format but we call it `price`, I feel like we should standardize. I have no preference on which word we use but given NIP-99 is already out there and in use, it might be easier to use that. 

https://github.com/nostr-protocol/nips/blob/master/99.md#price-examples